desc/compute/_omnigenity.py

```diff
@@ -346,6 +347,99 @@ def _f_T(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="eta",
+    label="\\eta",
+    units="rad",
+    units_long="radians",
+    description="Intermediate omnigenity coordinate along field lines",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="rtz",
+    data=[],
+    parameterization="desc.magnetic_fields.OmnigenousField",
+)
+def _eta(params, transforms, profiles, data, **kwargs):
+    # theta is used as a placeholder for eta (angle along field lines)
+    data["eta"] = (transforms["grid"].nodes[:, 1] - jnp.pi) / 2
+    return data
+
+
+@register_compute_fun(
+    name="alpha",
+    label="\\alpha",
+    units="rad",
+    units_long="radians",
+    description="Field line label, defined on [0, 2pi)",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="rtz",
+    data=[],
+    parameterization="desc.magnetic_fields.OmnigenousField",
+)
+def _alpha(params, transforms, profiles, data, **kwargs):
+    # zeta is used as a placeholder for alpha (field line label)
+    data["alpha"] = transforms["grid"].nodes[:, 2] * transforms["grid"].NFP
+    return data
+
+
+@register_compute_fun(
+    name="h",
+    label="h = \\theta + (N / M) \\zeta",
+    units="rad",
+    units_long="radians",
+    description="Omnigenity symmetry angle",
+    dim=1,
+    params=["x_lmn"],
+    transforms={"shift": [[0, 0, 0]]},
+    profiles=[],
+    coordinates="rtz",
+    data=["rho", "alpha", "eta"],
+    parameterization="desc.magnetic_fields.OmnigenousField",
+)
+def _omni_angle(params, transforms, profiles, data, **kwargs):
+    nodes = jnp.array([data["rho"], data["eta"], data["alpha"]]).T
+    data["h"] = (
+        transforms["shift"].basis.evaluate(nodes) @ params["x_lmn"]
```

Comment:
are the grids always NFP=1? as here you evaluate the shift on the alpha nodes which you had multiplied by NFP earlier, but it seems like you could just bake that into the basis instead?

Reply:
The grids always have the same NFP as the field/equilibrium. But alpha always needs to range from $\alpha \in [0, 2\pi)$ so we scale the NFP factor out in the compute function for alpha. The basis still uses multiples of NFP, but now I can't remember why...

Reply:
did we ever nail this down?